Microsoft.Toolkit.Uwp.Services/Services/OneDrive/OneDriveScopes.cs

```diff
@@ -47,6 +47,12 @@ public enum OneDriveScopes
         /// <summary>
         /// Grants read/write permission to a specific folder for your application
         /// </summary>
-        AppFolder = 0x04
+        AppFolder = 0x04,
+
+        /// <summary>
+        /// Single sign-in behavior .
+        /// </summary>
+        WlSignin = 0x05
```

Comment:
Add full set of enums for Core and Extended Scopes:  https://msdn.microsoft.com/en-us/library/office/dn631845.aspx